src/types/viewport.cpp

```diff
@@ -910,7 +910,7 @@ Viewport Viewport::ToOrigin() const noexcept
 }
 
 // Function Description:
-// - Creates a viewport from the intersection fo both the parameter viewports.
+// - Creates a viewport from the intersection for both the parameter viewports.

```

Comment:
I suspect this was meant to be "of" rather than "for".

Reply:
Ah of course. Besides, the previous function has a similar comment, with `of`.